### PR TITLE
Add new medals

### PR DESCRIPTION
**Pull request by AsikKEsel**
## About the PR

- [x] Adding the Cargo medal, engineer medal, medical meda, security medal, science medal, clown medal and case with all medals
**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, or this PR does not require an ingame showcase

**Changelog**

:cl:
- add: added Cargo medal, engineer medal, medical meda, security medal, science medal, case with all medals and... clown medal?

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Neck/medals.yml [hunk 1]
```diff
@@ -19,4 +19,70 @@
...
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/securitymedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/securitymedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckClownmedal
+  name: clown medal
+  description: Given for the best joke in the whole space. HONK!
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Clothing/Neck/medals.yml:83]: "the whole space" sounds wrong

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Comment by AsikKEsel** [comment]
What should be done to fix this error 1?

**Comment by deltanedas** [comment]
the medal case is size 5 but has 80 storage (infinite storage)
to solve this add something like
```
- type: Item
  size: 80
```

File: Resources/Prototypes/Entities/Clothing/Neck/medals.yml (hunk 1 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Clothing/Neck/medals.yml]: whats black and white and red all over? the mime and he died to blunt head trauma.

**Comment by BelovAlex2033** [comment]
wow its so good

**Pull request merged**